ios/FluentUI/Tab Bar/SideTabBar.swift

```diff
@@ -322,6 +322,14 @@ open class SideTabBar: UIView {
         return nil
     }
 
+    @objc public func itemView(with item: TabBarItem) -> UIView? {
```

Comment:
Can you add public documentation comments with the description of what this API does and making the precedence order clear? (searches the top section first and then the bottom one if not found)